packages/core/src/providers/42-school.ts

```diff
@@ -186,17 +186,17 @@ export interface FortyTwoProfile extends UserData, Record<string, any> {
  *
  * #### Configuration
  *```ts
- * import { Auth } from "@auth/core"
- * import FortyTwoSchool from "@auth/core/providers/42-school"
+ * import Auth from "@auth/core"
```

Comment:
Small note here, there is no default export, so the preexisting named export here was correct

Reply:
Sorry, my bad on the back and forward with the commits. Well spotted!
Thanks